clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -2386,4 +2399,60 @@ private void updateLastSeenEpochIfNewer(TopicPartition topicPartition, OffsetAnd
     String getClientId() {
         return clientId;
     }
+
+    static class KafkaConsumerMetrics {
```

Comment:
Could we move this to `consumer.internals`. Not sure there's a strong need for it to be internal